src/CognitoClient.php

```diff
@@ -259,6 +259,85 @@ public function deleteUser($accessToken)
         }
     }
 
+    
+    /**
+    * Disable a user
+    * @param string $username
+    */
+    public function adminEnableUser($username){
+        try {
+            $this->client->adminEnableUser([
+                'UserPoolId' => $this->userPoolId,
+                'Username' => $username
+            ]);
+        } catch (CognitoIdentityProviderException $e) {
+            throw CognitoResponseException::createFromCognitoException($e);
+        }
+    }
+    
+    /**
+    * Disable a user
+    * @param string $username
+    */
+    public function adminDisableUser($username){
+        try {
+            $this->client->adminDisableUser([
+                'UserPoolId' => $this->userPoolId,
+                'Username' => $username
+            ]);
+        } catch (CognitoIdentityProviderException $e) {
+            throw CognitoResponseException::createFromCognitoException($e);
+        }
+    }
+    
+    /**
+     * @param string $username
+     * @param string $password
+     * @param array $userAttributes
+     * @param boolean $confirmSignup
+     * @return object
+     * @throws Exception
+     */
+    public function adminCreateUser($username, $password, $attributes = [])
+    {
+        $userAttributes = $this->buildAttributesArray($attributes);
+        try {
+            $registeredUser = $this->client->adminCreateUser([
+                'UserPoolId' => $this->userPoolId,
+                'Username' => $username,
+                'TemporaryPassword' => $password,
+                'UserAttributes' => $userAttributes,
+                'MessageAction' => "SUPPRESS",
+                'DesiredDeliveryMediums' => ["EMAIL"]
+            ]);
+
+            return $registeredUser;
+        } catch (CognitoIdentityProviderException $e) {
+            throw CognitoResponseException::createFromCognitoException($e);
+        }
+    }
+
+
+    /**
+     * Set a admin added user as CONFIRMED. It sets the parameter password as final
+     * @param string $username
+     * @param string $password
+     * @return object
+     * @throws Exception
+     */
+    public function adminConfirmAddedUser($username, $password){
```

Comment:
Can you remove this method? I'm not sure it should belong in this class. You'd probably want it in your own service class.

Reply:
Can we just keep it? I am really needing this one for my project

Reply:
Sorry, but you should be able to just move this code to whatever part of your code calls this.